Illegal char in feautre file path

[//]: # (
. Note: for support questions, please use Stackoverflow or Gitter**. 
. This repository's issues are reserved for feature requests and bug reports.
.
. In case of any problems with Allure Jenkins plugin** please use the following repository 
. to create an issue: https://github.com/jenkinsci/allure-plugin/issues
.
. Make sure you have a clear name for your issue. The name should start with a capital 
. letter and no dot is required in the end of the sentence. An example of good issue names:
.
. - The report is broken in IE11
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?
When launching cucumber test with TestNG XML on newest version of allure-cucumber-jvm4 i get an InvalidPathException caused by Illegal char. E. g. of invalid path is file:src/test/resources/features/featurefile.feature

Illegal char is ":"

#### If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem


#### What is the expected behavior?
The file: preffix should not be added if it is possible

#### What is the motivation / use case for changing the behavior?


#### Please tell us about your environment:


| Allure version | 2.11.0 |
| --- | --- |
| Test framework | testng@6.14.3, Cucumber@4.3.1 |
| Allure integration Cucumber | allure-cucumber4-jvm@2.12.0 |
| Generate report using | allure-commandline@2.11.0 

#### Other information 
e.g of exception
[ERROR] Tests run: 15, Failures: 14, Errors: 0, Skipped: 1, Time elapsed: 8.483 s <<< FAILURE! - in TestSuite
[ERROR] runScenario(runners.LoginTestRunner)  Time elapsed: 0.01 s  <<< FAILURE!
java.nio.file.InvalidPathException: Illegal char <:> at index 4: file:src/test/resources/features/login/RMLogin.feature

[//]: # (
. e.g. detailed explanation, stacktraces, related issues, suggestions 
. how to fix, links for us to have more context, eg. Stackoverflow, Gitter etc
)

--- a/allure-cucumber4-jvm/src/main/java/io/qameta/allure/cucumber4jvm/LabelBuilder.java
+++ b/allure-cucumber4-jvm/src/main/java/io/qameta/allure/cucumber4jvm/LabelBuilder.java
@@ -24,6 +24,7 @@ import io.qameta.allure.util.ResultsUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.net.URI;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
@@ -162,7 +163,7 @@ class LabelBuilder {
     }
 
     private String featurePackage(final String uri, final String featureName) {
-        final Path parent = Paths.get(uri).getParent();
+        final Path parent = Paths.get(URI.create(uri).getSchemeSpecificPart()).getParent();
         if (Objects.nonNull(parent)) {
             final Stream<String> folders = StreamSupport.stream(parent.spliterator(), false)
                     .map(Path::toString);

